README.md

```diff
@@ -43,17 +43,28 @@ rails db:setup
 
 #### Developing with the Android app
 
-To run [simple-android](https://github.com/simpledotorg/simple-android/) app
-with the server running locally, you can use [ngrok](https://ngrok.com).
+To run [simple-android](https://github.com/simpledotorg/simple-android/) app with the server running locally, you can
+use [ngrok](https://ngrok.com).
 
 ```bash
 brew cask install ngrok
 rails server
 ngrok http 3000
 ```
 
-The output of the ngrok command will have an url that can be used to access
-local-server. This url should be set as `qaApiEndpoint` in gradle.properties.
+The output of the ngrok command is HTTP and HTTPS URLs that can be used to access your local server. Configure the
+following places with the HTTPS URL.
+
+In the `gradle.properties` file in the `simple-android` repository,
+```
+qaApiEndpoint=<HTTPS URL>
```

Comment:
This property does not currently exist in the [properties](https://github.com/simpledotorg/simple-android/blob/master/gradle.properties) file in simple-android. There are some other properties that kind of look like it, though 😄 .  Is this the right property and developers just need to add it? Do devs need to change `qaManifestEndpoint` or `qaFallbackApiEndpoint` ?

Reply:
I think the right property is indeed `qaManifestEndpoint` (it used to be called `qaApiEndpoint`) 👍 

For testing purposes, we shouldn't need to modify `qaFallbackApiEndpoint`, so we can leave that be.